app/src/main/res/layout/parcel_item.xml

```diff
@@ -153,6 +162,11 @@
 
                 </LinearLayout>
 
+                <TextView
+                    android:id="@+id/parcelLastMovementStatusTV"
```

Comment:
Rename to `parcelLastEventDateTV` (if renamed also in `CustomParcelsAdapterV2.java` as I suggested).